install/kubernetes/cilium/values.yaml

```diff
@@ -614,14 +614,16 @@ hubble:
   # -- Unix domain socket path to listen to when Hubble is enabled.
   socketPath: /var/run/cilium/hubble.sock
 
-  # -- An additional address for Hubble to listen to.
-  # Set this field ":4244" if you are enabling Hubble Relay, as it assumes that
+  # -- An additional port/address for Hubble to listen to.
+  # Set this field "4244" if you are enabling Hubble Relay, as it assumes that
   # Hubble is listening on port 4244.
-  listenAddress: ":4244"
-
+  listenPort: "4244"
```

Comment:
i'd keep `listenAddress` as is so that upgrading works without having to modify values.yaml file. 

either that, or add an entry in the upgrade guide, something similar to https://docs.cilium.io/en/v1.11/operations/upgrade/#helm-options

Reply:
Let's keep `listenAddress` please as it is always possible to extract the port programmatically if need be.